smart-contract/README.md

```diff
@@ -774,8 +827,24 @@ provenanced tx wasm execute \
     --testnet -o json | jq
 ```
 
-NOTE: Members can't burn another member's tokens. For example, `bank2` cannot burn the `bank1`
-tokens they currently hold.
+## Redeem and burn
+
+Let's say `bank2` wants to redeem `usdf.local` and burn `bank2.coin` at the same time.
+Members can redeem and burn their tokens in a single transaction.
+
+```bash
+provenanced tx wasm execute \
+    tp14hj2tavq8fpesdwxxcu44rty3hh90vhuz3ljwv \
+    '{"redeem_and_burn":{"amount":"2500"}}' \
+    --from bank2 \
+    --keyring-backend test \
+    --home build/node0 \
+    --chain-id chain-local \
+    --gas auto --gas-prices 1905nhash --gas-adjustment 2 \
+    --broadcast-mode block \
+    --yes \
+    --testnet -o json | jq
+```
```

Comment:
Seems like it would be valuable to show the balances again after this?

Reply:
added queries for before/after state